Sources/TuistSupport/Utils/FileHandler.swift

```diff
@@ -73,7 +73,7 @@ public protocol FileHandling: AnyObject {
     func changeExtension(path: AbsolutePath, to newExtension: String) throws -> AbsolutePath
     func resolveSymlinks(_ path: AbsolutePath) -> AbsolutePath
     func fileAttributes(at path: AbsolutePath) throws -> [FileAttributeKey: Any]
-    func filesAndDirectoriesContained(in path: AbsolutePath) -> [AbsolutePath]?
+    func filesAndDirectoriesContained(in path: AbsolutePath, shallow: Bool) -> [AbsolutePath]?
```

Comment:
how is `shallow` different from `contentsOfDirectory`? I'd suggest using that function and removing this parameter